src/RECONR/RECONR/src/call.hpp

```diff
@@ -0,0 +1,72 @@
+void operator()( const nlohmann::json& njoyArgs, 
```

Comment:
add overloaded call operator for modern interface

Reply:
There is a placeholder there now, but it doesn't really do anything right now.